README.md

```diff
@@ -52,6 +52,16 @@ git push heroku HEAD:master
 heroku open
 ```
 
+### Run Live Examples Locally
+To run *GoodData.UI Live Examples* locally:
+
+```bash
+git clone <this-repository>
```

Comment:
This is IMHO not necessary:
git clone <this-repository>
cd  <repository-folder>